.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Run linters and test
+
+on:
+  workflow_call:
+
+jobs:
+  lint-and-test:
+    name: Run linters and test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 💾
+        uses: actions/checkout@v3
+
+      - name: Setup node 🦠
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+
+      - name: Install dependencies 🔧
+        run: npm ci
+
+      - name: Run linters 📝
+        uses: wearerequired/lint-action@v2
+        with:
+          eslint: true
+          eslint_extensions: ts,tsx
+          prettier: true
+
+      - name: Build 🧬
+        run: npm run build
```

Comment:
Run linters and tests, but it looks like we also trigger build. I think build should be in separated file because if we would like to build&deply to skip waiting for linters we won't be able to achieve this.

Reply:
yup, copy-pastino style error, will change